app/src/main/java/com/example/mona/facebookoffline/EditActivity.java

```diff
@@ -41,15 +57,100 @@ public void onClick(View v) {
             public void onClick(View v) {
```

Comment:
Good call, it is getting pretty long. I can pull them out for now, and maybe later we can solve this problem using the Butterknife library!